pl_bolts/models/detection/retinanet/backbones.py

```diff
@@ -0,0 +1,41 @@
+from typing import Any, Optional
+
+import torch.nn as nn
+
+from pl_bolts.models.detection.components import create_torchvision_backbone
+from pl_bolts.utils import _TORCHVISION_AVAILABLE
+from pl_bolts.utils.warnings import warn_missing_pkg
+
+if _TORCHVISION_AVAILABLE:
+    from torchvision.models.detection.backbone_utils import resnet_fpn_backbone
+else:  # pragma: no cover
+    warn_missing_pkg("torchvision")
+
+
+def create_retinanet_backbone(
```

Comment:
This does code duplication.
There are Pros and Cons of both sides.

Reply:
This is exactly the same as the one in FasterRCNN, right? Why don't we define both of `create_retinanet_backbone` and `create_fasterrcnn_backbone` in somewhere in `pl_bolts/models/detection/components/` and use it in both RetinaNet and FasterRCNN?
https://github.com/PyTorchLightning/pytorch-lightning-bolts/blob/4e1c1502b70e4be59a9ac80878ec5ad5a212f87e/pl_bolts/models/detection/faster_rcnn/backbones.py#L15

Reply:
Same thoughts, I think it will be fine to refactor later. Though, I feel it keeps a little bit code clarity and keeps little code independency.